tests/integration/pilot/revisions/revisions_test.go

```diff
@@ -95,9 +95,10 @@ func TestMultiRevision(t *testing.T) {
 				}).
 				// tests bootstrap
 				WithConfig(echo.Config{
-					Service:   "vm",
-					Namespace: canary,
-					Ports:     []echo.Port{},
+					Service:    "vm",
+					Namespace:  canary,
+					DeployAsVM: true,
```

Comment:
oof

Reply:
I still feel like the test should fail. We use https://github.com/istio/istio/blob/master/samples/multicluster/expose-istiod.yaml which doesn't use the revisioned version. I'm running manually to verify